### ISSUE TITLE
Unable to resolve symbol: any? on (use 'overtone.live) Mac OSX 10.12.6

### ISSUE THREAD

**Issue by Chovin** [overtone/overtone#393]
I'm getting this when trying to run `(use 'overtone.live)`
Using MacOSX 10.12.6

`CompilerException java.lang.RuntimeException: Unable to resolve symbol: any? in this context, compiling:(overtone/sc/vbap.clj:183:3)`

Any ideas?

**Comment by hlolli** [comment]
Yes, this is a breaking change that I did, I did not want to break any code, any? is part of clojure 1.9
https://clojuredocs.org/clojure.core/any_q
maybe you are running on Clojure 1.8? Overtone project should be running on 1.9 so that you are getting this error makes me think "oh shit" :)

**Comment by Chovin** [comment]
Yup, I believe I am running on 1.8 as per [the instructions](https://github.com/overtone/overtone/blob/master/README.md#installation)

I'll try switching that out for 1.9 and see how it goes. Thank you!

**Comment by Chovin** [comment]
am one step closer!
updated my `project.clj` to have

```clojure
:dependencies [[org.clojure/clojure "1.9.0-alpha19"]
  	       [overtone "0.10.2"]])
```
Now I get this:
```clojure
user=> (use 'overtone.core)
nil
user=> (demo (sin-osc))

CompilerException java.lang.RuntimeException: Unable to resolve symbol: demo in this context, compiling:(/private/var/folders/db/jrl9f2rn4hd54s7tqw6578p00000gn/T/form-init7900775366574922470.clj:1:1)
```

**Comment by hlolli** [comment]
'overtone.core won't load overtone, 'overtone.live will. If you choose 'overtone.core, then you need to pass in (boot-internal-server) or (boot-external-server) to boot overtone.

**Comment by pmunozf** [comment]
I was running Overtone for the first time now, also having the 'any?' issue. I switched to clojure 1.9.0-alpha20 and now it works. Thanks!

**Comment by hlolli** [comment]
I will find an alternative to solve this 'any?' issue. Overtone should be compatible with older Clojure versions, this is entirely my fault :( but 1.9+ should work like you pointed out.

**Comment by Chovin** [comment]
also doing `(boot-internal-server)` doesn't work, but I guess I'll figure that out later.
`(use 'overtone.live)` works so long as I fix #394 as well 👍
